libpod/oci_util.go

```diff
@@ -28,7 +28,7 @@ type ociError struct {
 
 // Create systemd unit name for cgroup scopes
 func createUnitName(prefix string, name string) string {
-	return fmt.Sprintf("%s-%s.scope", prefix, name)
+	return fmt.Sprintf("%s-%s.scope/container", prefix, name)
```

Comment:
it is a crun specific feature.  It would not work with runc or another runtime.

I think the cgroup should be read from the process `/proc/PID/cgroup` file.